custom/conf/app.ini.sample

```diff
@@ -217,7 +217,7 @@ FILE_EXTENSIONS = .md,.markdown,.mdown,.mkd
 PROTOCOL = http
 DOMAIN = localhost
 ROOT_URL = %(PROTOCOL)s://%(DOMAIN)s:%(HTTP_PORT)s/
-; when STATIC_URL_PREFIX is empty it will follow APP_URL
+; when STATIC_URL_PREFIX is empty it will follow ROOT_URL
```

Comment:
Why following `ROOT_URL` instead?

Reply:
`ROOT_URL` is correct in config file context. The name `AppURL` is only used in code:

``` go
AppURL = sec.Key("ROOT_URL").MustString(defaultAppURL)